test/integration/params-validations-test.js

```diff
@@ -127,4 +127,22 @@ describe('params validations', () => {
       due_on: new Date('2012-10-09T23:39:01Z')
     })
   })
+
+  it.skip('Date is passed in correct format for notifications', () => {
```

Comment:
I wasn't able to get the test suite to pass, so I figured I would add the tests, then get some feedback and figure out where to put it

Reply:
I added this test here since the bug stems from old code in the validations. I feel like the proper place would be in a file where other activity endpoints are being tested, but I'm not sure where that is